### PR TITLE
Change structure of the project so we can support multiple sample app…

### PR DESCRIPTION
**Pull request by pierredelisle**
…lications in the future.

The only sample app we current have ("app") is renamed "3p-ecosystem" so that each sample app is given a proper descriptive name.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,104 +1,40 @@
...
-["Investigating Issues with the Sample App"](https://github.com/google-home/sample-app-for-matter-android/wiki/How-to-investigate-issues-with-the-sample-app-(GHSAFM)).
+See [3p-ecosystem sample app](3p-ecosystem/).
 
-## License
+## Clone the repository
 
-Google Home Sample App for Matter is licensed under
-the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
+```console
+$ git clone sso://user/pierred/google-matter-sampleapps-android
```
Comment (Vyrastas) [inline README.md:38]: Wrong cloning link

File: 3p-ecosystem/README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    Updated Git hooks.
+    Git LFS initialized.
+    ```
+
+3.  Once Git LFS is installed and initialized, you're ready to clone the
+    repository. When cloning completes, Git checks out the `main` branch
+    and downloads the native libraries for you.
+
+    ```console
+    $ git clone https://github.com/google-home/sample-app-for-matter-android.git
```
Comment (Vyrastas) [inline 3p-ecosystem/README.md:45]: sample-apps-for-matter-android

File: 3p-ecosystem/README.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+## Version
+
+Google Home Sample App for Matter follows the [Semantic](http://semver.org/)
+and [Android](https://developer.android.com/studio/publish/versioning) versioning guidelines for
+release cycle transparency and to maintain backwards compatibility.
+
+## Releases
+
+Always use the latest release as shown at
+https://github.com/google-home/sample-app-for-matter-android/releases
```
Comment (Vyrastas) [inline 3p-ecosystem/README.md:57]: app --> apps

File: 3p-ecosystem/README.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+* For an overview of the user interface and features, refer to
+    the [Google Home Sample App for Matter Guide](https://developers.home.google.com/samples/matter-app).
+* To review code samples and start building, refer to
+    the [Build an Android App for Matter](https://developers.home.google.com/codelabs/matter-sample-app)
+    Codelab.
+
+## Issues?
+
+If you run into problems with the sample app, please submit an issue on the 
+[Issues page](https://github.com/google-home/sample-app-for-matter-android/issues).
```
Comment (Vyrastas) [inline 3p-ecosystem/README.md:88]: app --> apps

File: 3p-ecosystem/README.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+It is always greatly appreciated if you can try to reproduce the issue you encountered 
+from a clean state as described in document
+["Investigating Issues with the Sample App"](https://github.com/google-home/sample-app-for-matter-android/wiki/How-to-investigate-issues-with-the-sample-app-(GHSAFM)).
```
Comment (Vyrastas) [inline 3p-ecosystem/README.md:92]: Should we update the Wiki to mention 3p-ecosystem etc as well?

app --> apps in the link too

**Review by Vyrastas — CHANGES_REQUESTED** [review]

File: 3p-ecosystem/README.md (hunk 2 above)
Comment (pierredelisle) [reply 3p-ecosystem/README.md]: Done.

File: README.md (hunk 1 above)
Comment (pierredelisle) [reply README.md]: Done.

File: 3p-ecosystem/README.md (hunk 3 above)
Comment (pierredelisle) [reply 3p-ecosystem/README.md]: Done.

File: 3p-ecosystem/README.md (hunk 4 above)
Comment (pierredelisle) [reply 3p-ecosystem/README.md]: Done.

**Review by Vyrastas — APPROVED** [review]

**Pull request merged**